DemoqaProject/tests/ElementsTest/ButtonsTest.cs

```diff
@@ -0,0 +1,48 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class ButtonsTest : BaseSeleniumTest
+    {
+        [Test]
+        public void CheckDoubleClick()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
+            Buttons buttons = new Buttons();
+            buttons.NavigateToButtons();
+            
+            Thread.Sleep(2000);
```

Comment:
Here I am not waiting for specific element. I am waiting just loading of web page. Cause website render is not working quickly.

Reply:
There is no problem to add a method that would wait for page load. It could be done by waiting for some specific element to be displayed or more complicated way to wait for page finishes rendering.

Reply:
done